app/views/users/show.html.erb

```diff
@@ -1,16 +1,22 @@
-<h1> Details of: <%=@user.name%> </h1>
-<p><%= @user.bio%></p>
+<ul class="pt-3">
+      <div class="container border rounded-3 p-3 col-8 justify-content-center">
+        <div class="d-inline ">
+          <img src= '<%= @user.photo %>' width="100" height="100" class="" />
+        </div>
+        <h3 class="d-inline ps-5"><%= @user.name %></h3>
+        <p class="text-end">Number of Posts: <%=@user.post_counter%></p>
+      </div>
+      <div class="container border rounded-3 p-3 col-8 justify-content-center mt-4"><%=@user.bio%></div>
+    </ul>
 
-
-<h1>All posts by  <%= @user.name %></h1>
-<p>
   <% @posts = Post.where(user_id: @user.id) %>
 </p>
 <ul>
     <% @posts.each do |post| %>
```

Comment:
- [ ] According to the requirements, you are supposed to use the `recent-posts` method in your user's model to show the 3 most recent posts. Here, you are showing all posts made by a user. Please fix that 👌 .